### ISSUE TITLE
RPlidar A1M8 Firmware Update

### ISSUE THREAD

**Issue by hsynklceng** [Slamtec/rplidar_sdk#91]
Hi,
I have 2 lidar and one lidar's firmware is 1.28 and another lidar's firmware is 1.27. I want to lidars' firmware update to newest version. But i can not find anywhere how can i update.
Can you help me for firmware update?

**Comment by AustinDeric** [comment]
yes i would like this too.  Does @tony-slamtec or anyone else actively support this sdk or product?

**Comment by dreamer15th** [comment]
same problem, the newest firmware is no longer available on their website
